### PR TITLE
New powershell connector (PSv2)

### PR DESCRIPTION
**Pull request by rhouthuijzen**
tested at custommer environment
Readme is based on the new template
The connector is not based on the new template

### CODE REVIEW

**Comment by rschouten97** [comment]
@rhouthuijzen the code appears to be well-structured. However, since we are focusing on the PS v2 template, please ensure that the code aligns with the template standards.
I recommend waiting at least a week before making any significant changes, as the template is still undergoing revisions following its recent release.
Furthermore, I recommend merging this pull request only once we have actually implemented this connector.

File: fieldMapping.json [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+      ]
+    },
+    {
+      "Name": "UsId",
+      "Description": "[Mandatory] Gebruiker code",
+      "Type": "Text",
+      "MappingActions": [
+        {
+          "MapForActions": [
+            "Update"
```
Comment (Rick-Jongbloed) [inline fieldMapping.json:313]: Create also required: https://github.com/Tools4everBV/HelloID-Conn-Prov-Target-AFAS-Profit-Users/issues/15

**Comment by Rick-Jongbloed** [comment]
Please add a notion in de readme that relation number the UsId complex script should be updated.

**Comment by Rick-Jongbloed** [comment]
Please make sure insite can be set and unset at resp enable & disable

The following code should be added
```
    if ($account.PSObject.Properties.Name -Contains 'InSi') {
        $updateAccountFields += "InSi"
    }
```

**Comment by rhouthuijzen** [comment]
> Please add a notion in de readme that relation number the UsId complex script should be updated.

Done

**Comment by rhouthuijzen** [comment]
> Please make sure insite can be set and unset at resp enable & disable
> 
> The following code should be added
> 
> ```
>     if ($account.PSObject.Properties.Name -Contains 'InSi') {
>         $updateAccountFields += "InSi"
>     }
> ```

Before we add this. We should also make the $previousAccount list dynamic. This needs to be tested first.

**Review by JeroenBL — APPROVED** [review]
Based on what I've seen in the changed/added code, everything is OK. So ready to merge.

**Pull request merged**